src/components/BRB/BRBMainComponent.tsx

```diff
@@ -522,6 +524,10 @@ const ButtonItem = styled(Button)`
 
 const ButtonBar = styled(Button)`
   letter-spacing: 0.03em;
+  font-family: Glancyr;
```

Comment:
Fallback font should be mentioned: https://developer.mozilla.org/en-US/docs/Web/CSS/font-family